holmes/core/tool_calling_llm.py

```
@@ -286,6 +286,125 @@ def __init__(
+                    # Execute the tool
+                    tool_result = self._invoke_tool(
+                        tool_to_call=mock_tool_call,
+                        previous_tool_calls=[],
+                        trace_span=DummySpan(),
+                        tool_number=None,
+                    )
+
+                    # Add tool result to messages
+                    messages.append(tool_result.as_tool_call_message())
+
+                except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)